.github/workflows/build.yml

```diff
@@ -118,11 +114,9 @@ jobs:
 
     - name: Setup .NET
       uses: actions/setup-dotnet@v4
-      with:
-        dotnet-version: 9.0.x
 
     - name: Setup WSL2
-      uses: Vampire/setup-wsl@v5
+      uses: Vampire/setup-wsl@f40fb59d850112c9a292b0218bca8271305b9127 # v5.0.0
```

Comment:
GH actions security seems to be in the news a lot recently so I pinned the third-party action. AFAIU dependabot can still update it to a new commit and show the diff